edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.MqttBrokerAdapter/upstream/BrokeredCloudConnection.cs

```diff
@@ -4,26 +4,58 @@ namespace Microsoft.Azure.Devices.Edge.Hub.MqttBrokerAdapter
     using System;
     using System.Threading.Tasks;
     using Microsoft.Azure.Devices.Edge.Hub.Core.Cloud;
+    using Microsoft.Azure.Devices.Edge.Hub.Core.Identity;
     using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Extensions.Logging;
 
     public class BrokeredCloudConnection : ICloudConnection
     {
+        IIdentity identity;
+
         public BrokeredCloudConnection(BrokeredCloudProxy cloudProxy)
```

Comment:
Why is this not a ICloudProxy? We should avoid passing in actual types.. if possible.